docs/ExportFormats.md

```diff
@@ -0,0 +1,26 @@
+# Export Formats
+
+## Summary table CSV export
+
+The summary table CSV export includes the following columns:
+
+* `Phyloreference ID`: A URL or URI (absolute, relative or local) that identifies this phyloreference.
+  * Example `#Alligatoridae`
+* `Label`: A label used to identify this phyloreference.
+  * Example: `Alligatoridae`
+* `Type`: The type of this phyloreference.
+  * May be one of the following values:
+    * `Apomorphy-based clade definition`
+    * `Maximum clade definition`
+    * `Minimum clade definition`
+    * `Invalid definition (...)` (with the reason for the invalid definition in brackets)
+* `Definition`: The clade definition as entered in the `Definition in free-form text`.
+  * Example: `Last common ancestor of Crocodylus niloticus, Osteolaemus tetraspis, and Tomistoma schlegelii and all of its descendents.`
+* `Internal specifier N`: A description of each internal specifier (_N_ is a number starting from 1).
+  * Example: `Caiman crocodilus`
+* `External specifier N`: A description of each external specifier (_N_ is a number starting from 1).
+  * Example: `Caiman crocodilus`
+* `Phylogeny: [Phylogeny label] expected`: The node resolved for this phylogeny (_N_ is a number starting from 1, and _Phylogeny label_ is the label of this phylogeny).
+  * Example: `Alligatoridae`
```

Comment:
This isn't compliant with the current item format description?

Reply:
It's supposed to be the label of the node on the specified phylogeny where this phyloreference is expected to resolve -- so if phyloref #Alligatorinae is expected to resolve at the node "Alligatoridae" on the phylogeny "Fig 1 from Brochu 2003", the column will be named "Phylogeny: Fig 1 from Brochu 2003 expected" and will have the value "Alligatoridae". Does this make sense, or do I need to clarify something?